### PR TITLE
docs(ootbc): introduce AWS DynamoDB connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

Introduce a new AWS DynamoDB Connector

## Are there related marketing activities

TBD.

## When should this change go live?

TBD

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This section gives an overview of the **out-of-the-box Connectors** available in
 ## Outbound Connectors
 
 - [Automation Anywhere Connector](automation-anywhere.md) - Orchestrate your [Automation Anywhere](https://www.automationanywhere.com/) queue from your BPMN process.
+- [Amazon DynamoDB Connector](aws-dynamodb.md) - TThe DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```
Comment (sbuettner) [inline docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:12]:
```suggestion
- [Amazon DynamoDB Connector](aws-dynamodb.md) - The DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```

File: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ This section gives an overview of the **out-of-the-box Connectors** available in
 ## Outbound Connectors
 
 - [Automation Anywhere Connector](automation-anywhere.md) - Orchestrate your [Automation Anywhere](https://www.automationanywhere.com/) queue from your BPMN process.
+- [Amazon DynamoDB Connector](aws-dynamodb.md) - TThe DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```
Comment (sbuettner) [inline versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:12]:
```suggestion
- [Amazon DynamoDB Connector](aws-dynamodb.md) - The DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```

File: versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ This section gives an overview of the **out-of-the-box Connectors** available in
 ## Outbound Connectors
 
 - [Automation Anywhere Connector](automation-anywhere.md) - Orchestrate your [Automation Anywhere](https://www.automationanywhere.com/) queue from your BPMN process.
+- [Amazon DynamoDB Connector](aws-dynamodb.md) - TThe DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```
Comment (sbuettner) [inline versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:12]:
```suggestion
- [Amazon DynamoDB Connector](aws-dynamodb.md) - The DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```

File: versioned_docs/version-8.2/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ This section gives an overview of the **out-of-the-box Connectors** available in
 ## Outbound Connectors
 
 - [Automation Anywhere Connector](automation-anywhere.md) - Orchestrate your [Automation Anywhere](https://www.automationanywhere.com/) queue from your BPMN process.
+- [Amazon DynamoDB Connector](aws-dynamodb.md) - TThe DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```
Comment (sbuettner) [inline versioned_docs/version-8.2/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:12]:
```suggestion
- [Amazon DynamoDB Connector](aws-dynamodb.md) - The DynamoDB Connector allows you to interact with [Amazon DynamoDB NoSQL database service](https://aws.amazon.com/dynamodb/) within your BPMN process, enabling you to store and retrieve data from tables, as well as perform queries and scans.
```

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit 👍

**Pull request merged**